.env

```diff
@@ -1,17 +1,42 @@
 SCHOLARWORKS_HOST=localhost
-REGISTRY_HOST=registry.gitlab.com
-REGISTRY_URI=/notch8/cal-state-hyrax
 UPLOAD_BASE=./tmp
-POSTGRES_DB=csum_postgres_development
-POSTGRES_HOST=localhost
-POSTGRES_PASSWORD=DatabaseFTW
-POSTGRES_USER=postgres
-REDIS_SERVER=localhost
-DOCKER_PORT=3001:3001
-HOST=0.0.0.0
-
-HANDLE_SERVER=54.244.77.182:8000
-HS_ADMIN=300:0.NA/20.500.12680
-HS_PRIVATE_KEY=admpriv.pem
-HS_PREFIX=20.500.12680
+# DOCKER_PORT=3001:3001
+# HOST=0.0.0.0
+# HANDLE_SERVER=54.244.77.182:8000
```

Comment:
I think we need to uncomment these as they are needed for a gem they are using
https://github.com/csuscholarworks/handle-system

Reply:
also, not the biggest deal but we should keep the vars alphabetized